code/game/machinery/doors/door.dm

```diff
@@ -191,6 +192,7 @@ obj/machinery/door/proc/try_to_crowbar(obj/item/I, mob/user)
 	if(prob(40/severity))
 		if(secondsElectrified == 0)
 			secondsElectrified = -1
+			shockedby += "\[[time_stamp()]\]EM Pulse"

```

Comment:
we can't really find the origin of the EMP, but at least we known it's been emp'ed.